docs-src/content/functions/crypto.yml

```diff
@@ -103,6 +103,52 @@ funcs:
       - |
         $ gomplate -i '{{ "hello world" | crypto.EncryptAES "swordfish" 128 | base64.Encode }}'
         MnRutHovsh/9JN3YrJtBVjZtI6xXZh33bCQS2iZ4SDI=
+  - name: crypto.ECDSAGenerateKey
+    experimental: true
+    description: |
+      Generate a new Elliptic Curve Private Key and output in
+      PEM-encoded PKCS#1 ASN.1 DER form.
+
+      Go's standard NIST P-224, P-256, P-384, and P-521 elliptic curves are all
+      supported.
+
+      Default curve is P-256 and can be overridden with the optional `curve`
+      parameter.
+    pipeline: true
+    arguments:
+      - name: curve
+        required: false
+        description: |
+          One of Go's standard NIST curves, P-224, P-256, P-384, or P-521 -
+          defaults to P-256.
+    examples:
+      - |
+        $ gomplate -i '{{ crypto.ECDSAGenerateKey }}'
+        -----BEGIN EC PRIVATE KEY-----
+        ...
+      - |
+        $ gomplate -i '{{ $key := crypto.ECDSAGenerateKey "P-521" -}}
+          {{ $pub := crypto.ECDSADerivePublicKey $key -}}'
+        hello
```

Comment:
No need for this example (plus `hello` as an output doesn't make sense)

```suggestion
```

Reply:
That example is "adapted" from the `RSAGenerateKey` example, so maybe you want to remove the "hello" from there as well?

Reply:
The difference is that the `RSAGenerateKey` example  also adds an encrypt & decrypt, so that output makes sense in that context.